solidity/dashboard/src/services/liquidity-rewards.js

```diff
@@ -159,6 +163,46 @@ class UniswapLPRewards extends LiquidityRewards {
 
     return this._calculateAPY(r, WEEKS_IN_YEAR)
   }
+
+  /**
+   * Calculates lp token balance for the given pair
+   * The calculations were done based on
+   * https://uniswap.org/docs/v2/advanced-topics/understanding-returns/#why-is-my-liquidity-worth-less-than-i-put-in
+   *
+   * @param {string} lpBalance Balance of liquidity token for a given uniswap pair deposited in
+   * the LPRewards` contract.
+   * @return {Promise<{token0: string, token1: string}>}
+   */
+  calculateLPTokenBalance = async (lpBalance) => {
+    const pairData = await getPairData(this.wrappedTokenAddress.toLowerCase())
+
+    const shareOfUniswapPool = percentageOf(
+      lpBalance.toString(),
+      fromTokenUnit(pairData.totalSupply).toString()
```

Comment:
why do we convert `pairData.totalSupply` to Wei only total supply, but not other values?